CBLClient/Apps/CBLTestServer-iOS/CBLTestServer-iOS/Server/ReplicatorConfigurationRequestHandler.swift

```diff
@@ -62,6 +135,8 @@ public class ReplicatorConfigurationRequestHandler {
             let heartbeat: String? = args.get(name: "heartbeat")
             let maxRetries: String? = args.get(name: "max_retries")
             let maxRetryWaitTime: String? = args.get(name: "max_timeout")
+            let collections: [Collection]? = args.get(name: "collections")
+            let collection_configuration: [CollectionConfiguration]? = args.get(name: "configuration")
```

Comment:
I think this switch-case needs to refactor and separate out, it has already become ~170 lines. It will become harder to debug/understand each logic.

Reply:
Please elaborate on it.
Steps required to optimize it.

Reply:
since its already separated, collections, collection-configs are not necessary to pass through here right?